pkg/k8sapi/discovery.go

```diff
@@ -113,7 +115,7 @@ func (d *Controller) GetCNIPods() []string {
 		cniPods = append(cniPods, k)
 	}
 
-	log.Info("GetCNIPods discovered", cniPods)
+	log.Infof("GetCNIPods discovered %v", cniPods)
```

Comment:
Could you remove the `log.Infof()` and `fmt.Printf()` on lines 94 and 95 in `discovery.go` as well?